packages/mdl-drawer/_mixins.scss

```diff
@@ -0,0 +1,152 @@
+/**
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@import "./functions";
+@import "mdl-theme/mixins";
+@import "mdl-typography/mixins";
+
+/* All variables are local, so disable pattern checking. */
+/* stylelint-disable scss/dollar-variable-pattern */
+
+@mixin mdl-drawer-dir_($property, $value) {
+  $directions: (
+    ltr: (
+      "start": "left",
+      "end": "right"
+    ),
+    rtl: (
+      "start": "right",
+      "end": "left"
+    )
+  );
+
+  #{mdl-drawer-replace($property, map-get($directions, ltr))}: $value;
+
+  [dir="rtl"] & {
+    #{mdl-drawer-replace($property, map-get($directions, ltr))}: auto;
+    #{mdl-drawer-replace($property, map-get($directions, rtl))}: $value;
+  }
+}
+
+@mixin mdl-drawer-toolbar-spacer_() {
+  &__toolbar-spacer {
+    display: flex;
+    position: relative;
+    flex-direction: row;
+    flex-shrink: 0;
+    align-items: flex-center;
+    height: 56px;
```

Comment:
Is the idea to eventually replace this and the value within the media query with variables from `mdl-toolbar` once we have it?